### ISSUE TITLE
DM wording weirdness

### ISSUE THREAD

**Issue by ara4n** [element-hq/element-web#15733]
<img width="297" alt="Screenshot 2020-11-16 at 09 45 22" src="https://user-images.githubusercontent.com/1294269/99237512-7e53fd80-27f0-11eb-8aae-030b32e01f30.png">

on the plus side, we're no longer calling it a room!

On the minus side, do we really want to call it a "DM" rather than "Direct Message"?
Also, the "...invited Matthew" is completely redundant given it's a DM, i think, and should be collapsed into the room creation summary?

**Comment by weeman1337** [comment]
> On the minus side, do we really want to call it a "DM" rather than "Direct Message"?

From an user perspective, what is "direct message" at all? For me this is a single message but this is kind of a chat room.

Shouldn't it rather be something like "This is the beginning of the conversation of XYZ and you."?

**Comment by niquewoodhouse** [comment]
For fuller context, a user currently sees this on web, I think.

![image](https://user-images.githubusercontent.com/1587166/113690274-2183b180-96c3-11eb-8d01-23f6d0594c76.png)

> Also, the "...invited Matthew" is completely redundant given it's a DM, i think, and should be collapsed into the room creation summary?

I agree.  I'm not personally sure why these two bullet points (You created this DM + You invited XX) are needed at all in the DMs screen as everything is already said in the larger text summary above, but I have heard its a really important thing for some people/use cases.   I suppose you need something to be able to expand/collapse the room summary, but I again personally don't understand the need for the summary, I just know some people find it useful - there is no summary on Android for DMs, but there is for web and iOS.

> ...should be collapsed into the room creation summary?

The room creation summary = the expand/collapse list of things that have happened? It's currently a bit inconsistent with the pre-expanded notes (one says person joined the room, the other says person created this DM) and there's a lot of 'room' references in there that go against the idea of not calling a DM/chat a room.   There is no room summary on Android for DMs?

![image](https://user-images.githubusercontent.com/1587166/113705438-57319600-96d5-11eb-8d96-051e961b3ed4.png)

We could change the terms to be room/DM agnostic. 

- person joined the room = person joined
- person created the room =  person started this conversation (I call it conversation because it's a term used in the intro copy, like direct message - either works I think as they've already been used, I just personally lean towards conversation)
- person made the room invite only = person made this invite only
- person made future room history visible to all room members = person made future messages visible by everyone invited
- `person has allowed guests to join the room` = Is this needed?  It's not present in iOS DM summary and is a bit scary for a private conversation.
- person joined the room = person joined


________



> From a user perspective, what is "direct message" at all? For me this is a single message but this is kind of a chat room.

That's a really interesting question.  Direct message seems to mean peer:peer(s) communication not necessarily around a specific topic, whereas a room is more topic driven, I _think_.   Looking at other products I see some using direct messages as a heading, which suggests it does make _some_ sense to their users, so it has some precedence.  

Discord:

![image](https://user-images.githubusercontent.com/1587166/113710228-5b60b200-96db-11eb-9fc2-e39d5e7889b9.png)

Slack: 

![image](https://user-images.githubusercontent.com/1587166/113710712-f8234f80-96db-11eb-8b15-b7ce6c1b1db9.png)

**Comment by vertigo220** [comment]
This was just part of a lot of confusion my friend and I, both new to matrix, ran across. Not really sure who did what, but when we first started, we somehow ended up with a room between the two of us instead of a DM. This was additionally confusing because the room name was his name for me and my name for him, and when I went into the room settings, we were both admins and I could change my role, indicating I had control, but I couldn't change his, so I didn't have control...

Anyways, we started a DM, but then it showed the above "user created this **DM**" and "user joined the **room**" which further added to the confusion of it all. Then he tried to forget the original room and it didn't do anything despite multiple attempts. I chalk a lot of this up to us just not knowing what we're doing, but ideally people should be able to use it without knowing what they're doing and be able to easily make sense of things and not have conflicting terminology, and it should have probably been easier to start a DM in the first place rather than accidentally creating a room instead, though unfortunately I don't know how/why that happened.
